exp/views/user.py

```diff
@@ -101,7 +101,7 @@ class ResearcherListView(LoginRequiredMixin, DjangoPermissionRequiredMixin, gene
     '''
     template_name = 'accounts/researcher_list.html'
     # TODO needs to change once oauth in
-    queryset = User.objects.filter(demographics__isnull=True, is_active=True)
```

Comment:
Speaking of "demographics__isnull" which is not really relevant for distinguishing between participant/researcher any longer, some of the querysets in the API are still using this filter.